iphone/Maps/UI/Search/MWMSearchManager.h

```diff
@@ -2,6 +2,8 @@
 #import "MWMSearchManagerObserver.h"
 #import "MWMSearchManagerState.h"
 
+static CGFloat const kWidthForiPad = 320.0;
```

Comment:
This will create several copies of the same constant in each mm file that includes this header.

You may try the following:
```
// Use namespace to avoid polluting the global one.
namespace searchmanager {
CGFloat constexpr kWidthForiPad = 320.0;
}
```
and then use `searchmanager::kWidthForiPad` in the code.